lib/src/facts/windows/operating_system_resolver.cc

```diff
@@ -100,8 +100,8 @@ namespace facter { namespace facts { namespace windows {
         // Override default release with Windows release names
         auto version = result.release.substr(0, lastDot);
         bool consumerrel = (wmi::get(vals, wmi::producttype) == "1");
-        if (version == "6.4") {
```

Comment:
Does anyone know of any reason we should keep the 6.4 in here? According to everything I found, this version doesn't exist; it was replaced in favor of 10.0. I tried this on both a Windows 10 and a 2016 box, and both returned 10.0.x as the kernel version.